pkg/minishift/provisioner/minishift_detector_test.go

```diff
@@ -71,7 +71,7 @@ VARIANT_VERSION="1.0.0-alpha.1"
 
 	osRelease, _ := provisioner.GetOsReleaseInfo()
 	if osRelease.Variant != "minishift" {
-		t.Fatal("Release info should contain 'minishift' variant")
+		t.Fatal("Release info must contain 'minishift' variant")
```

Comment:
generally we try to avoid using the word "should", from the understanding that something is either in or out. Should is ambiguious, evasive, and human-associated, plus it doesn't translate well. In this case it's quite simple, something was supposed to be there but it isn't. If the error message appears when you miss a required piece of data, then that piece of data must be there.